### ISSUE TITLE
Error sound being played when pressing keys on macOS

### ISSUE THREAD

**Issue by martinet101** [r0x0r/pywebview#566]
Hi, i was creating a calculator with windows, and it was working without any problems. But then, when I tried to run it on macOS, when I press any key (My js code accepts keyboard capture), it works, but it plays classical "Funk" sound that is played when you are trying to do something not allowed

The sound is this one: [https://www.youtube.com/watch?v=8rz44o_gGpE](https://www.youtube.com/watch?v=8rz44o_gGpE)

I'm using Python 3.7.6
on macOS 10.15.5
pywebview 3.2

**Comment by r0x0r** [comment]
No idea.
It could be something with your code. A minimal example would help.

**Comment by martinet101** [comment]
I don't think so, because if I open the same html on chrome the sound does
not "appear" ...

El dg., 12 de jul. 2020, 23:40, Roman <notifications@github.com> va
escriure:

**Comment by LTSENG01** [comment]
I'm having a similar issue where I'm trying to detect a button press (Enter key) using JS. It plays the system "Funk" sound. It is not within an input field or form.

```
$("#id").keyup((e) => {
    if (e.key === "Enter") {
        console.log("Enter pressed.");
    }
});
```

--
Python 3.8.3
macOS 10.15.6
pywebview 3.3.2

**Comment by r0x0r** [comment]
Ok I tracked down the problem. I pushed an experimental fix to the master branch, would you verify if it works for you?

**Comment by LTSENG01** [comment]
I copied the change to my installation of the library, and it does work. Thanks!

**Comment by r0x0r** [comment]
Released in 3.3.3
